packages/terra-form-select/tests/nightwatch/select/SelectTests.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import React from 'react';
+import { Link } from 'react-router';
+
+const SelectTests = () => (
+  <div>
+    <ul>
+      <li><Link to="/tests/select-tests/default">Select - Default</Link></li>
+      <li><Link to="/tests/select-tests/disabled">Select - Disabled</Link></li>
+      <li><Link to="/tests/select-tests/invalid">Select - Invalid</Link></li>
+      <li><Link to="/tests/select-tests/long-text">Select - Long Text</Link></li>
+      <li><Link to="/tests/select-tests/controlled">Select - Controlled</Link></li>
+    </ul>
```

Comment:
It might be good to add a test that has a preselected option for a controlled select.

Reply:
Also a test with select within a modal to verify onRequestFocust/onReleaseFocus

Reply:
The controlled select test does have a preselected option. 
Added jest tests for requestFocus and releaseFocus, since those are just passed through to the popup component and select doesn't do anything with them: https://github.com/cerner/terra-core/pull/978/commits/70812139c85b71e4035c0388406c73685de32e1f